### DISCUSSION TITLE
Unable to find stock config directory

### DISCUSSION THREAD

**Discussion by JuttDB** [netdata/netdata#15636] [Q&A] [answered]
Hello,
          i am using docker. i would like to change to config using edit-config.

Here is script for docker 
`sudo docker run -d --name=netdata \
  --pid=host \
  --network=host \
  --hostname=my_docker_netdata \
  -v $(pwd)/netdataconfig/netdata:/etc/netdata \
  -v netdatalib:/var/lib/netdata \
  -v netdatacache:/var/cache/netdata \
  -v /etc/passwd:/host/etc/passwd:ro \
  -v /etc/group:/host/etc/group:ro \
  -v /proc:/host/proc:ro \
  -v /sys:/host/sys:ro \
  -v /etc/os-release:/host/etc/os-release:ro \
  -v /var/run/docker.sock:/var/run/docker.sock:ro \
  --restart unless-stopped \
  --cap-add SYS_PTRACE \
  --cap-add SYS_ADMIN \
  --security-opt apparmor=unconfined \
  netdata/netdata`

Here is output from my host machine 

![image](https://github.com/netdata/netdata/assets/20758400/f29df72f-2b23-47d7-92d2-35f636851e56)

edit-config is "Unable to find stock config directory"

Any assistance you can provide would be greatly appreciated.

Thank you

**Comment by ilyam8** [comment]
Hi, @JuttDB. This problem is fixed in https://github.com/netdata/netdata/pull/15641

**Reply by JuttDB** [reply]
**new image is pulled today so it should be fine.**

`Ubuntux@ubuntu:~/netdataconfig/netdata$ ./edit-config list

Autodetected containerized Netdata instance. Attempting to autodetect container ID.
ERROR: Could not autodetect container ID. It must be supplied on the command line with the --container option.

**According to above error.  --container  is added with container id** 

Ubuntux@ubuntu:~/netdataconfig/netdata$ ./edit-config list --container 9daf1ebe67eb49d8f8d64a
ERROR: Unrecognized option list.`

**Reply by ilyam8** [reply, marked as answer]
Did you check the error?

> Ubuntux@ubuntu:~/netdataconfig/netdata$ ./edit-config **list**
> ERROR: Unrecognized option **list**.`
